test/system/505-networking-pasta.bats

```diff
@@ -789,3 +774,14 @@ EOF
     CONTAINERS_CONF_OVERRIDE=$containersconf run_podman run --net=pasta:--ns-mac-addr,"$mac2" $IMAGE ip link show myname
     assert "$output" =~ "$mac2" "mac address from cli is set on custom interface"
 }
+
+### Rootless unshare testins
+
+@test "Podman unshare --rootless-netns with Pasta" {
+    skip_if_remote "unshare is local-only"
+
+    pasta_iface=$(default_ifname)
```

Comment:
Again, I really dislike the use of `pasta` in the variable name (and test name) because one day there will be a different default. I would much prefer `rootless` and/or an explicit `skip if this is not pasta`.

Reply:
> one day there will be a different default.

**Never**. It's pasta _e basta_.